### DISCUSSION TITLE
🔴 Scheduled maintenance affecting Mac mini M1 availability on January 27

### DISCUSSION THREAD

**Discussion by helinanever** [codemagic-ci-cd/codemagic-docs#2335] [Announcements]
Our data center is carrying out maintenance that will affect the availability of our Apple M1 machines (`mac_mini_m1`). The maintenance window is from 12 PM UTC to 3 PM UTC (7 AM ET to 10 AM ET) on January 27.

During this time, we will be operating with a reduced number of M1 build machines. As a result, you may be experiencing longer than usual build queues.

Apple M2 and Mac Pro, Linux, and Windows machine availability will not be affected.

Thank you for your understanding.

See also: https://status.codemagic.io/

**Comment by mikhail-tokarev** [comment]
The planned maintenance was completed, and all services resumed operation as expected.
